File: app/src/main/java/org/demo/weatherapp/view/BindingAdapter.java
package org.demo.weatherapp.view;

import android.content.Context;
import android.graphics.Typeface;

public class BindingAdapter {

    private BindingAdapter() {
        //do nothing
    }

    public static Typeface setWeatherFont(Context context) {
        return Typeface.createFromAsset(context.getAssets(), "fonts/weathericons-regular-webfont.ttf");
    }
}

File: app/src/main/java/org/demo/weatherapp/view/BindingAdapter.kt
package org.demo.weatherapp.view

import android.content.Context
import android.graphics.Typeface

class BindingAdapter {

    companion object {

        @JvmStatic
        fun setWeatherFont(context: Context): Typeface {
            return Typeface.createFromAsset(context.assets, "fonts/weathericons-regular-webfont.ttf")
        }
    }
}
